src/VisualStudio/Core/Def/Implementation/LanguageService/AbstractLanguageService`2.VsCodeWindowManager.cs

```diff
@@ -60,6 +88,14 @@ private void OnOptionChanged(object sender, OptionChangedEventArgs e)
                 AddOrRemoveDropdown(enabled);
             }
 
+            private WorkspaceRegistration GetWorkspaceRegistration()
+            {
+                ErrorHandler.ThrowOnFailure(_codeWindow.GetBuffer(out var buffer));
+
+                var textContainer = _languageService.EditorAdaptersFactoryService.GetDataBuffer(buffer).AsTextContainer();
```

Comment:
What happens if GetDataBuffer returns null? I can't think of a way it might but sometimes that *can* happen.